mod_ci/models.py

```diff
@@ -83,3 +83,17 @@ def __repr__(self):
         :rtype str(platform, status): str
         """
         return '<Platform {p}, maintenance {status}>'.format(p=self.platform.description, status=self.disabled)
+
+
+class blackisted_users(Base):
+    __tablename__ = 'BlockedUsers'
+    # user_id is the user's id GitHub assigns.
+    user_id = Column(Integer, primary_key=True)
+    # username refers to the username of the user.
+    username = Column(String(41), unique=True)
```

Comment:
Why do we need this?

Reply:
Because if we only have user id it'll be confusing. The addition of Username and Name serve for better categorizing and finding users easier

Reply:
But those can be retrieved by using the GitHub API... If you store them and a user changes name, the information is outdated.

Reply:
If you want I can only leave username and delete name?

Reply:
The userid will not change but if it's the same id and the username or name is different change the record